ramls/loan.json

```
@@ -113,6 +113,15 @@
       "pattern": "^[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{12}$",
       "type": "string"
     },
+    "billedLostItemFlag": {
+      "description": "Indicates if the aged to lost fee has been billed (for use where delayed billing is set up)",
+      "type": "boolean"
+    },
+    "dateBillLostItem": {
```

Is it possible to rename it?
```suggestion
    "billLostItemDate": {
```